fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1453,6 +1453,7 @@ public List<List<String>> getSingleTranInfo(long dbId, long txnId) throws Analys
                 for (Long tblId : tblIds) {
                     Table tbl = db.getTable(tblId);
                     if (tbl != null) {
+                        // won't check privilege in new RBAC framework
```

Comment:
Please add in comments why “won't check privilege in new RBAC framework”

Reply:
It's a decesion made by pm after discussion, mainly because it won't reveal any key information, i think there is no need to put the reason here. For any details, please refer to this doc: https://starrocks.feishu.cn/docx/doxcnYps0N29zynkPPzi7IcOTAn